deploy/grafana-dashboard/availability_row.py

```diff
@@ -0,0 +1,51 @@
+from grafanalib.core import RowPanel, BarGauge, GridPos, TimeSeries, ORIENTATION_VERTICAL, \
+    GAUGE_DISPLAY_MODE_BASIC
+from grafanalib.influxdb import InfluxDBTarget
+
+from util import read_query
+
+prewarming_pool_size = BarGauge(
+    title="Prewarming Pool Size",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("prewarming-pool-size"))],
+    gridPos=GridPos(h=10, w=11, x=0, y=3),
+    allValues=True,
+    orientation=ORIENTATION_VERTICAL,
+    displayMode=GAUGE_DISPLAY_MODE_BASIC,
+    max=None,
+)
+
+idle_runner = TimeSeries(
+    title="Idle Runner",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("idle-runner"))],
+    gridPos=GridPos(h=10, w=13, x=11, y=3),
+    lineInterpolation="stepAfter",
+)
+
+runner_startup_duration = TimeSeries(
+    title="Runner startup duration",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("runner-startup-duration"))],
+    gridPos=GridPos(h=10, w=12, x=0, y=13),
+    unit="ns",
+)
+
+used_runner = TimeSeries(
+    title="Used Runner",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("used-runner"))],
+    gridPos=GridPos(h=10, w=12, x=12, y=13),
+)
+
+availability_row = RowPanel(
+    title="Availability",
+    collapsed=True,
```

Comment:
Should we generally open all rows? I am unsure...

Reply:
I have left them collapsed, because otherwise the panels can not be defined nested in the RowPanels anymore (all panels would be listed directly in the Dashboard definition). I am fine with both..

Reply:
Ah, I didn't know that. Having that collapsed is fine; we might just want to uncollapse them after saving a new version (I find it easer to look at the uncollapsed version, but the code looks better with the nested definitions).

Reply:
I also like the uncollapsed version more and I think with [14c4212](https://github.com/openHPI/poseidon/pull/200/commits/14c4212bfb7dc259866ae774d7197071a467aab1) the code quality is not that much influenced..
PS. I changed the order of the rows because I find the availability panels the most interesting?!

Reply:
Awesome, I just saw that when rebasing (and testing) my branch. Thanks for having another look and working on this solution; I think it is still clean and nice.

And yes, the changed order might be useful to get a good impression on the first glance 🔍